modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -1385,7 +1519,8 @@ private void _assertObjectAction(
 
 	private void _assertWebhookObjectAction(
 			String firstName, String objectActionTriggerKey,
-			String originalFirstName, int status)
+			ObjectDefinition objectDefinition, String originalFirstName,
+			int status)
```

Comment:
I saw that `_assertWebhookObjectAction` is used a several times. So, we can create a new `_assertWebhookObjectAction` method to that it is not necessary change the current uses. Something like this:

```java
private void _assertWebhookObjectAction(
	String firstName, String objectActionTriggerKey,
	String originalFirstName, int status)
throws Exception {

      _assertWebhookObjectAction(
	firstName, objectActionTriggerKey,
	_objectDefinition, originalFirstName,
	status)
}
```
What do you thing?